src/Utils/BlockTemplateUtils.php

```diff
@@ -635,11 +635,18 @@ function( $template ) use ( $customised_template_slugs ) {
 
 	/**
 	 * Returns whether the blockified templates should be used or not.
-	 * If the option is not stored on the db, we need to check if the current theme is a block one or not.
+	 * First, we need to make sure WordPress version is higher than 6.1 (lowest that supports Products block).
+	 * Then, if the option is not stored on the db, we need to check if the current theme is a block one or not.
 	 *
 	 * @return boolean
 	 */
 	public static function should_use_blockified_product_grid_templates() {
+		$minimum_wp_version = '6.1.1';
```

Comment:
I don't think it's a big deal, but asking out of curiosity: when registering the Products block [we use version `6.1`](https://github.com/woocommerce/woocommerce-blocks/blob/0984663b66514be1bb028b8daf5efcb63d31faa6/assets/js/blocks/product-query/index.tsx#L44) (without the last `.1`). Is it intentional that here we are using `6.1.1` instead?

Reply:
Good point, thanks for this! 🙇‍♂️ 

It's my bad, I checked the mechanism [in here](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/woocommerce-gutenberg-products-block.php#L21), got stuck during testing and I missed to update the version.

It's fixed in [this commit](https://github.com/woocommerce/woocommerce-blocks/pull/8588/commits/6e4690576cbe6696476bd794b5e3c5fed0305bdb). 🙏

Reply:
> EDIT: I updated the PR description to reflect the change as well.

It might be good to update the PR title as well. :upside_down_face: